### DISCUSSION TITLE
Vertical drawing of text

### DISCUSSION THREAD

**Discussion by Kiritsu** [SixLabors/ImageSharp.Drawing#249] [Q&A] [answered]
Hello,

I'd like to draw text on an image but with the particularity that the text is oriented vertically:

![image](https://user-images.githubusercontent.com/22667054/203523805-c864d410-0840-40d5-8af4-c79b0be6f548.png)

What would be the proper way to do this? Right now, I'd just do a multi-line drawing with a single character per line, but I'm not sure it is the right way.

Thank you!

**Comment by JimBobSquarePants** [comment, marked as answer]
You want [`TextOptions`](https://docs.sixlabors.com/api/Fonts/SixLabors.Fonts.TextOptions.html#SixLabors_Fonts_TextOptions_LayoutMode) [`LayoutMode`](https://docs.sixlabors.com/api/Fonts/SixLabors.Fonts.LayoutMode.html).

**Reply by Kiritsu** [reply]
Oh indeed, I swear I only had seen the Left-Right Right-Left. I probably had an old version of the lib. Thank you

**Reply by JimBobSquarePants** [reply]
Must be very old!
